test/e2e/common_suite_test.go

```diff
@@ -870,3 +874,253 @@ func doTestCreatePeerPodWithAuthenticatedImageWithoutCredentials(t *testing.T, a
 	pod := newPod(namespace, podName, podName, imageName, withRestartPolicy(v1.RestartPolicyNever))
 	newTestCase(t, "InvalidAuthImagePeerPod", assert, "Peer pod with Authenticated Image with Invalid Credentials has been created").withPod(pod).withAuthenticatedImage().withAuthImageStatus(expectedAuthStatus).withCustomPodState(v1.PodPending).run()
 }
+
+func doTestCaaDaemonsetUpgrade(t *testing.T) {
```

Comment:
We talked about to split the test cases as the common_suit_test.go grows greatly, maybe this is right PR to add a separate file.